nemo/core/neural_types/neural_type.py

```diff
@@ -21,14 +21,15 @@
     'NeuralTypeError',
     'NeuralPortNameMismatchError',
     'NeuralPortNmTensorMismatchError',
-    'CanNotInferResultNeuralType',
```

Comment:
why is this deleted?

Reply:
Because there is no code that tries to "infer" the neural type.